frontend.rst

```diff
@@ -25,14 +25,11 @@ Requires a build step?            yes                no
 Works in all browsers?            yes                yes
 Supports `Stimulus/UX`_           yes                yes
 Supports Sass/Tailwind            yes                :ref:`yes <asset-mapper-tailwind>`
-Supports React, Vue, Svelte?      yes                yes :ref:`[1] <note-1>`
-Supports TypeScript               yes                no :ref:`[1] <note-1>`
+Supports React, Vue, Svelte?      yes                yes :ref:`[1] <ux-note-1>`
+Supports TypeScript               yes                no :ref:`[1] <ux-note-1>`
 ================================  =================  ======================================================
 
-Notes
-~~~~~
-
-.. _note-1:
+.. _ux-note-1:
```

Comment:
The `ref` wasn't unique (we use the same ref in the BC page), so the above links were pointing THERE instead of here.